tools/build/src/devfile/meta-yaml-to-devfile-yaml.ts

```diff
@@ -90,11 +90,19 @@ export class MetaYamlToDevfileYaml {
       return;
     }
 
+    let name = '';
+    if (metaYaml.name) {
+      name = metaYaml.name;
+      name = name.replace(/\//gi, '-');
```

Comment:
This will convert all slashes `/` to dashes... but we still might end up with really long workspace prefixes like `ws-skeleton/jupyter/5.7.0` --> `ws-skeleton-jupyter-5.7.0`

Reply:
Suggestion below should remove the first segment, and "che-" prefixes, then change /next or /insiders to next or insiders, and finally remove the last /version segment.

Reply:
tested on https://playcode.io/javascript with:
```
named = "";
named="eclipse/che-theia/7.55.0";
named = named.replace(/[^/]+\//, '').replace(/che-/,'').replace(/\/(next|insiders)/,'$1').replace(/\/[^/]+/,'');
console.log(named)

named="ws-skeleton/jupyter/5.7.0"
named = named.replace(/[^/]+\//, '').replace(/che-/,'').replace(/\/(next|insiders)/,'$1').replace(/\/[^/]+/,'');
console.log(named)

named="eclipse/che-theia/next";
named = named.replace(/[^/]+\//, '').replace(/che-/,'').replace(/\/(next|insiders)/,'$1').replace(/\/[^/]+/,'');
console.log(named)

named="che-incubator/che-code/insiders"
named = named.replace(/[^/]+\//, '').replace(/che-/,'').replace(/\/(next|insiders)/,'$1').replace(/\/[^/]+/,'');
console.log(named)

named="che-incubator/che-idea/next"
named = named.replace(/[^/]+\//, '').replace(/che-/,'').replace(/\/(next|insiders)/,'$1').replace(/\/[^/]+/,'');
console.log(named)
```
...

```
theia
jupyter
theianext
codeinsiders
ideanext
```

Reply:
With my changes the workspace prefixes are not long:
**ws-skeleton/jupyter/latest** -->  name: **jupyter** 
**che-incubator/che-code/insiders** ---> name: **che-code**

Reply:
how will we differentiate che-code (latest release, 7.55.0) from che-code-insiders? there are TWO versions of this editor available upstream, right?

Reply:
Dashboard reads index.json from the plugins reg https://eclipse-che.github.io/che-plugin-registry/main/index.json and takes displayName, name and version. For example for che-code we have:
```
{
    "id": "che-incubator/che-code/insiders",
    "description": "Code OSS integration in Eclipse Che.",
    "displayName": "che-code",
    "links": {
      "devfile": "/v3/plugins/che-incubator/che-code/insiders/devfile.yaml"
    },
    "name": "che-code",
    "publisher": "che-incubator",
    "type": "Che Editor",
    "version": "insiders"
  },
  {
    "id": "che-incubator/che-code/latest",
    "description": "Code OSS integration in Eclipse Che.",
    "displayName": "che-code",
    "links": {
      "devfile": "/v3/plugins/che-incubator/che-code/latest/devfile.yaml"
    },
    "name": "che-code",
    "publisher": "che-incubator",
    "type": "Che Editor",
    "version": "latest"
  },
```